sled-agent/src/sim/server.rs

```diff
@@ -177,221 +171,43 @@ impl Server {
                 .await;
         }
 
+        Ok(Server {
+            config: config.clone(),
+            log: log.clone(),
+            sled_agent,
+            http_server,
+            pantry_server: None,
+        })
+    }
+
+    /// Starts the pantry service and add it to the DNS config builder
```

Comment:
Is this comment still accurate?  I don't see where we add it to the DNS config builder.

Reply:
Good catch, the DNS config was separated from this invocation. Updated in e340a0439